traefik_kop.go

```diff
@@ -204,3 +209,22 @@ func getContainerPort(dockerClient client.APIClient, conf *dynamic.Configuration
 	logrus.Debugf("overriding service port from container host-port: using %s (was %s) for %s", exposedPort, port, svcName)
 	return exposedPort
 }
+
+func getContainerIP(dockerClient client.APIClient, conf *dynamic.Configuration, svcType string, svcName string) string {
+	svcName = strings.TrimSuffix(svcName, "@docker")
+	routerName := getRouterOfService(conf, svcName, svcType)
+	routerName = strings.TrimSuffix(routerName, "@docker")
+
+	logrus.Debugf("found router %s for service %s", routerName, svcName) // dedup
+	container, _ := findContainerByServiceName(dockerClient, svcType, svcName, routerName)
+
+	if isNetworkSet(container) {
+		network_name := container.Config.Labels["traefik.docker.network"]
```

Comment:
Can probably just remove the `isNetworkSet` method as it isn't used elsewhere and just keep the below empty string check. It will have the same effect of checking if a network name is available.